### DISCUSSION TITLE
dedicated binding that only completes filenames

### DISCUSSION THREAD

**Discussion by aheays** [fish-shell/fish-shell#9901] [Ideas]
Sometimes completion definitions prevent completion of filenames.  When these definitions are incomplete or outdated and a filename is appropriate it can be quite hard to enter it.  I could find no command dedicate to filename/path completion to bind to a key.  This would certainly be useful at time.

**Comment by faho** [comment]
Our position here is basically that that completion script has a bug and should be fixed.

You can do that yourself, by copying it to ~/.config/fish/completions/ (from the original location, typically /usr/share/fish/completions/foo.fish for a command called "foo").

I don't think we want to add a workaround like this.
